package.json

```diff
@@ -9,6 +9,7 @@
     }
   ],
   "scripts": {
+    "start": "gulp dev",
```

Comment:
You're right that's an error with the version of nodejs.

The error I was seeing was that the `gulp` command simply can't be found when I try to set it up on a fresh system. It does seem like an older version of gulp or gulp-cli may have added a wrapper to `/usr/local/bin/gulp` that makes everything work if you don't do the install on a fresh system.